### ISSUE TITLE
Error when JSON object has a nulled property

### ISSUE THREAD

**Issue by amunhoz** [Microsoft/napajs#180]
I think it's a minor problem, but took me 1 hour to discover the cause.

Test code:

```
var napa = require('napajs');
var zone = napa.zone.create('sample-zone', { workers: 4 });


function test (obj) {
  obj.prop = null;
  console.log(JSON.stringify(obj));
  return obj;
}
zone.broadcast(test.toString());
var result = zone.execute('','test', [{propa:1, propb:3, propc:null}])
```

The error ocurred first time when returning a object from napa, wich i could not repllicate.  But it is throwing the same error when you send a object with null property too.

**Comment by helloshuangzi** [comment]
Thanks for your feedback, @amunhoz. I just made #181 to fix this bug.

**Comment by helloshuangzi** [comment]
Close this issue. It is resolved by #181.
